Shared/Services/LoopringGraphQLService.cs

```diff
@@ -818,7 +818,92 @@ public LoopringGraphQLService()
                 return null;
             }
         }
+        public async Task<int> GetAccountTotalNfts(string accountId, CancellationToken cancellationToken = default)
```

Comment:
We need a unit test for that one. I already have one locally, let me know if you want me to push it.

Reply:
Yeah, go ahead 😊 i'm back at the computer tomorrow morning

Reply:
OK, I pushed it. Please have a look if you agree on the way I did it.

Reply:
Looks nice :) Don't need any more complicated tests than that. I would like to add tests to checks by adding them to workflow, but that can be done later.

Reply:
> I would like to add tests to checks by adding them to workflow

Not sure I understand, but the tests are automatically run, when a) anything is merged into dev and/or master and b) whenever I push something to any of the branches in my fork. The latter I had to add myself, so you'd need to do the same for your fork, if you want.

Reply:
If I'm doing a PR that's already the case, since the branch resides in my fork until it is merged. If you want the same, you need to add the workflow action to your fork (preferably on your master branch, so we don't pull it in).

Reply:
But this is another PR essentially... just added for bringing it up

Reply:
I think we've got a misunderstanding here, but as you say, it's a separate issue, so please create one and we can discuss it there.

Reply:
I just pushed another commit for the test, as it actually would not have detected if the total would be smaller than it actually is (because it only tried to get as many as total claimed there would be).

Reply:
I'm not sure if I understood it correctly. GetAccountNFTs(0, nftCount, accountID) did not work as expected in the test?

Reply:
Checked, and tests are Ok, but I don't know if it's necessary with +1 when getting the actual nfts.

Reply:
No, everything was working fine - just the test could have been potentially been tricked.

If *hypothetically* `GetAccountTotalNfts` would return a smaller number, the test would not have detected this error, because it only tried to get as many NFTs as it was told. Now it tries to get one more, and if there actually are more, it would detect this!

Reply:
Ahhh, I see.